test/Npgsql.Tests/ConnectionTests.cs

```diff
@@ -635,7 +635,7 @@ public void BreakConnectorInPool(bool keepAlive)
                 else
                 {
                     Assert.That(conn.ProcessID, Is.EqualTo(connectorId));
-                    Assert.That(() => conn.ExecuteScalar("SELECT 1"), Throws.Exception.TypeOf<NpgsqlException>());
+                    Assert.That(() => conn.ExecuteScalar("SELECT 1"), Throws.Exception.AssignableTo<NpgsqlException>());
```

Comment:
Not sure what changed, but the thrown exception is now a `PostgresException`.

Reply:
Probably some race condition triggered by different environmental conditions on Github Actions. Change looks fine.